src/store/getters.ts

```diff
@@ -623,7 +619,7 @@ export const getLocationOptions = (state: StateIF): Array<any> => {
   if (getRequestActionCd(state) === RequestCode.MVE) {
     return Locations.filter(location => location.value === Location.BC)
   }
-  return Locations // return all locations
+  return Locations.filter(() => true)
```

Comment:
This fixes a separate issue where the jurisdiction menu is not set (see ticket). I determined that the reason for this is that the old code used the original Locations object, and it was being overwritten (possibly by the v-select). The new code creates a copy of the object (using code consistent within this method. I know there are other ways to do this but this seemed the more clear).